app/helpers/blacklight/catalog_helper_behavior.rb

```diff
@@ -0,0 +1,276 @@
+# -*- encoding : utf-8 -*-
```

Comment:
Sorry, I was looking at the wrong file. I have now taken the necessary methods from the file and removed the rest of it.